src/api/aptosConfig.ts

```diff
@@ -8,13 +8,29 @@ import { AptosApiType, DEFAULT_NETWORK } from "../utils/const";
 
 /**
  * This class holds the config information for the SDK client instance.
+ *
+ * @example To use Aptos Labs endpoints, simply set the `network` property to the desired network.
+ * ```typescript
+ *   const config = new AptosConfig({ network: Network.DEVNET });
+ * ```
+ *
+ * @example To use custom endpoints, set the `network` property to the desired network, and the appropriate URLs. The URL
+ * must contain the full path and will not have a `/v1` added.
+ * ```typescript
+ *   const config = new AptosConfig({
+ *     network: Network.TESTNET,
+ *     fullnode: "https://my-testnet-fullnode.com/v1",
```

Comment:
this needs to be better documented. We also use `Network.CUSTOM` where the sdk expects to fullnode,faucet and indexer custom endpoints.
https://github.com/aptos-labs/aptos-ts-sdk/blob/main/src/api/aptosConfig.ts#L57